common/src/main/java/com/revenuecat/purchases/common/HTTPClient.kt

```diff
@@ -20,9 +24,11 @@ import java.io.OutputStreamWriter
 import java.net.HttpURLConnection
 import java.net.MalformedURLException
 import java.net.URL
+import kotlin.jvm.Throws
```

Comment:
is this unused?

Reply:
I asked myself the same question, but it's used. It's used in line https://github.com/RevenueCat/purchases-android/blob/601fcc2ae32d095da1295123dbfb70223a8b3737/common/src/main/java/com/revenuecat/purchases/common/HTTPClient.kt#L42

I don't know how it got added, but I just looked into it a bit more and realized that before this import we were using a typealias in `Throws.kt`

```
public actual typealias Throws = kotlin.jvm.Throws
```

So it's the same to use the typealias or importing it from `kotlin.jvm`. I will revert the import